lib/node_modules/@stdlib/fs/resolve-parent-paths/lib/async.js

```diff
@@ -284,17 +283,17 @@ function each( paths, dir, done ) {
 	var count;
 	var child;
 	var spath;
-	var FLG = 0; // flag to track if we are done traversing a directory level
+	var FLG = false; // flag to check if we are done resolving all paths
 	var out;
 	var i;
 
 	count = 0;
-	out = filled( null, paths.length );
+	out = [];
```

Comment:
You should still initialize the array to a fixed length. Doing `out[ idx ]`, as you do below, can put the array into object mode, thus killing perf for downstream users wanting to iterate over the array.